completions/bluetoothctl

```diff
@@ -0,0 +1,46 @@
+# bluetoothctl(1) completion                                    -*- shell-script -*-
+
+_bluetoothctl()
+{
+    local cur=${COMP_WORDS[COMP_CWORD]}
+    local prev=${COMP_WORDS[COMP_CWORD-1]}
```

Comment:
If you intend to use the auto-loading mechanism of bash-completion (i.e., putting the file in the `completions` directory and letting `bash-completion` load it automatically on demand), I suggest using `_init_completion` for the initialization (as done in other completions). It doesn't only provide a better extraction of the words but also the contextual completions for the variable names and redirections. It might also be used by `bash-completion` for hooks in the future.